clusters/app.ci/grafana-loki/admin_grafana.yaml

```diff
@@ -0,0 +1,109 @@
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: grafana-loki
+---
+kind: Group
+apiVersion: v1
+metadata:
+  name: grafana-loki-admins
+users:
+- vrutkovs
+- periklis
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: grafana-loki-admin
+  namespace: grafana-loki
+subjects:
+- kind: Group
+  apiGroup: rbac.authorization.k8s.io
+  name: grafana-loki-admins
+roleRef:
+  kind: Role
+  apiGroup: rbac.authorization.k8s.io
+  name: admin
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: grafana
+  namespace: grafana-loki
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: grafana
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: grafana
+    spec:
+      volumes:
+        - name: grafana-config
+          secret:
+            secretName: grafana-config
+            defaultMode: 420
+        - name: grafana-datasources
+          secret:
+            secretName: grafana-datasources
+            defaultMode: 420
+      containers:
+        - name: grafana-sso
+          image: 'docker.io/grafana/grafana:7.3.2'
+          ports:
+            - containerPort: 3000
+              protocol: TCP
+          resources: {}
+          volumeMounts:
+            - name: grafana-config
+              mountPath: /etc/grafana
+            - name: grafana-datasources
+              mountPath: /etc/grafana/provisioning/datasources
+      restartPolicy: Always
+      terminationGracePeriodSeconds: 30
+      dnsPolicy: ClusterFirst
+      securityContext: {}
+      schedulerName: default-scheduler
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 25%
+      maxSurge: 25%
+  revisionHistoryLimit: 10
+  progressDeadlineSeconds: 600
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: grafana
+  namespace: grafana-loki
+spec:
+  ports:
+    - protocol: TCP
+      port: 3000
+      targetPort: 3000
+  selector:
+    app: grafana
+  type: ClusterIP
+---
+kind: Route
+apiVersion: route.openshift.io/v1
+metadata:
+  name: grafana
+  namespace: grafana-loki
+spec:
+  host: grafana-loki.ci.openshift.org
```

Comment:
@hongkailiu do we have a doc for how to get custom domains on app.ci for your ingress?

Reply:
Reused ci-search / release-controller method